recipe/meta.yaml

```diff
@@ -8,20 +8,19 @@ source:
   - url: https://raw.githubusercontent.com/python/typeshed/main/LICENSE
     sha256: 295f8538c94ae5c3043301cf7cff1c852dab6a786a8ddee471e061b40d5ecabe
   - url: https://pypi.io/packages/source/t/types-setuptools/types-setuptools-{{ version }}.tar.gz
-    sha256: 07648088bc2cbf0f2745107d394e619ba2a747f68a5904e6e4089c0cb8322065
+    sha256: 22ad498cb585b22ce8c97ada1fccdf294a2e0dd7dc984a28535a84ea82f45b3f
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: true # [py<36]
+  script: {{ PYTHON }} -m pip install --no-deps --no-build-isolation . -vv
 
 requirements:
   host:
     - pip
-    - python >=3.6
+    - python
   run:
-    - python >=3.6
-    - types-docutils
+    - python
 
 test:
   commands:
```

Comment:
```suggestion
  requires:
    - pip
  imports:
    - setuptools
  commands:
    - pip check
```

Reply:
What does importing the python module `setuptools` accomplish in a stubs package?

Additionally we have `setuptools` as a dependency on our python3.12 package and its included by default in earlier python versions....

Reply:
right setuptools will come from pip (not python)